problems/unhappy_clients.md

```diff
@@ -0,0 +1,100 @@
+---
+slug: unhappy-clients
+stages:
+  - development
+short_description: How to deal with unhappy clients? First, stay calm. No matter how good you are at what you do, sooner or later you'll come across an unhappy client. Navigate to the article to read more. 
+tags:
+  - Unhappy Clients
+  - communication
+  - feedback
+  - problem identification
+  - extra mile
+keywords:
+  - client
+  - communication
+  - complaint
+  - planning
+  - methodology
+---
+
+# Unhappy Clients
+
+## TL;DR
+
+No matter how good you are at what you do, sooner or later you'll come across an unhappy client. Although your ego can get hurt, don't be disheartened. Bad feedback often leads to great work, as you learn valuable lessons and really push your skills.
+
+So, what to do in such a situation?
+
+![](/files/keep_calm.png)
+
+[Keep Calm and Carry On *(thanks to Ministry of Information – UK, 1939)*](/files/keep_calm.png)
```

Comment:
link should navigate e.g. to dxheroes.io website if it is your image

discussed with @ridlees

Reply:
It's not my image. It was created by British Ministry of Information in 1939. Internet is full of these images, I do not remember where exactly I found this one. Entering the link to dxheroes.io.